### PR TITLE
Yarn!

### PR DESCRIPTION
**Pull request by chrisnojima**
```
brew install yarn
```

```
cd desktop
yarn
``` 

@keybase/react-hackers

### CODE REVIEW

**Comment by chrisnojima** [comment]
- [ ] Remove shrinkwrap from build process

**Comment by chrisnojima** [comment]
- [ ] Get yarn on build machines

**Comment by gabriel** [comment]
I just brew installed yarn on the macos build machine

File: Jenkinsfile [hunk 1]
```diff
@@ -236,11 +233,10 @@ helpers.rootLinuxNode(env, {
                                     wrap([$class: 'Xvfb']) {
                                         println "Test Windows JS"
                                         dir("visdiff") {
-                                            bat "npm install"
+                                            bat "yarn"
                                         }
-                                        bat "npm install .\\visdiff"
                                         dir("desktop") {
-                                            bat "npm run vendor-install"
```
Comment (chromakode) [inline Jenkinsfile]: :sheep: Will need to do something here for electron, different from vendor-install. I'll look into this.

File: packaging/desktop/package_darwin.sh [hunk 2]
```diff
@@ -149,8 +149,8 @@ get_deps() {(
 package_electron() {(
   cd "$client_dir/desktop"
 
-  ../packaging/npm_mess.sh
```
Comment (chromakode) [inline packaging/desktop/package_darwin.sh]: :pray:

Comment (chrisnojima) [inline packaging/desktop/package_darwin.sh]: this is mostly a 'hopeful' mutation. we might need yarn_mess.sh :)

**Comment by chrisnojima** [comment]
@chromakode we need to also talk about flow being in the js-vendor

**Comment by chrisnojima** [comment]
@keybase/react-hackers CI passes now! Now we just need to get the bots to build a test build. can i get more 👀  on this pr

**Pull request merged**